src/Nox.Solution/Models/Domain/Entities/Relationships/EntityRelationship.cs

```diff
@@ -32,6 +33,8 @@ public class EntityRelationship : DefinitionBase
     [Description("The name of the target entity that this entity relates to.")]
     public string Entity { get; internal set; } = null!;
 
+    public TypeUserInterface? UserInterface { get; internal set; }
```

Comment:
Just a suggestion to make TypeUserInterface? UserInterface non nullable => public TypeUserInterface UserInterface { get; internal set; } = new TypeUserInterface()
 
And therefore avoid null checks, and spreading out default values everywhere?